migrations/20231105104353-create-food-category.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('food_categories', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        defaultValue: Sequelize.literal('uuid_generate_v4()'),
+        type: Sequelize.UUID,
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      restaurant_id: {
```

Comment:
why have you kept restaurant id in category?

Reply:
Because one restaurant can have many category

Reply:
Do I need to remove the one to many relationship between restaurants and food_categories table?

Reply:
category belongs to food not restaurant

Reply:
for example there is a restaurant an it have a starter category and there are many menu items which are in this starter category and it might be possible that some restaurants don't have starter as food_category. Due to this reason, I have made this kind of relationship between these tables.